### PR TITLE
Use isEmpty to trigger default init queries

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

1 use isEmpty instead show for the init SQLs
2 add job desc for SQL tab of Spark UI
3 make it simple.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @pan3793 @turboFei @ulysses-you

File: kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala [hunk 1]
```diff
@@ -35,9 +35,9 @@ object Utils extends Logging {
 
   import org.apache.kyuubi.config.KyuubiConf._
 
-  def strToSeq(s: String): Seq[String] = {
+  def strToSeq(s: String, sp: String = ","): Seq[String] = {
```
Comment (pan3793) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala:38]: what if sql contains `,` ?

File: externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala [hunk 2]
```diff
@@ -53,20 +55,15 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
...
-          .split(";")
-          .filter(_.trim.nonEmpty)
-          .foreach { sql =>
-            info(s"Execute session initializing sql: $sql")
-            sparkSession.sql(sql).show
-          }
+        val ss = spark.newSession()
+        this.conf.get(ENGINE_SESSION_INITIALIZE_SQL).foreach { sqlStr =>
+          ss.sparkContext.setJobGroup(handle.identifier.toString, sqlStr, interruptOnCancel = true)
+          ss.sql(sqlStr).isEmpty
```
Comment (pan3793) [inline externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala:64]: can we keep log here?

Comment (yaooqinn) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala:64]: the UI is more straightforward

**Review by pan3793 — APPROVED** [review]
